scripts/misc/docker_pull_baseimage.py

```diff
@@ -14,7 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 from runpy import run_module
+import sys
+
+try:
+    from ros_buildfarm import __version__  # noqa: F401
+except ImportError:
+    sys.path.insert(
+        0, os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
```

Comment:
If this were a novel technique I would want it to have an explanatory comment like
```suggestion
# If the ros_buildfarm is not on the current PYTHONPATH add it using the current script path as an anchor.
try:
    from ros_buildfarm import __version__  # noqa: F401
except ImportError:
    sys.path.insert(
        0, os.path.dirname(os.path.dirname(os.path.dirname(__file__))))
```

but it doesn't make much sense to add it here when it appears elsewhere or add it everywhere that it appears.